app/views/posts/show.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>Posts#show</h1>
-<p>Find me in app/views/posts/show.html.erb</p>
+<h1>Post Details</h1>
+ <div class='bio font'>
+    <h4>Post:<%= @post.id %> by <%= @post.user.name %></h4> 
+     <div class='padd'>
+        <h5>Comments:<%= @post.comments_counter %></h5>
+        <h5>Likes:<%= @post.likes_counter %></h5>
+     </div>
+    <h4><%= @post.text %></h4>
+    <hr/>
+    <% @post.recent_comment.each do |comment| %>
+        <h5><%= comment.user.name %>: <%= comment.text %> </h5>
+    <% end %>
+</div>
```

Comment:
- [ ] Post show page should display the post with all comments listed as shown on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=ABIFLA6OIXRW5GF52AJGR33BHIMGO). Kindly display all comments and not only the recent on the show page.

Reply:
Already implemented it! Thanks for the feedback! smile